PROJECT

```diff
@@ -12,32 +12,41 @@ resources:
   domain: kubeflow.org
   group: pipelines
   kind: Pipeline
-  path: github.com/sky-uk/kfp-operator/apis/pipelines/v1
```

Comment:
This has not been updated correctly in the previous version change. New versions should always be introduced using the `kubebuilder` CLI